include/mesh.tcc

```diff
@@ -1746,99 +1746,24 @@ bool mpm::Mesh<Tdim>::assign_nodal_concentrated_forces(
   return status;
 }
 
-//! Assign velocity constraints to nodes
```

Comment:
Why are these removed?

Reply:
Because they are not part of the mesh.tcc file anymore. They are in the load_bcs folder as part of the constraints.tcc file. This was removed as a merge and resolve merge conflicts commits when I merged from develop.